### ISSUE TITLE
dndhelper quit unexpectedly

### ISSUE THREAD

**Issue by AceAttorneyMaster111** [ChordPro/chordpro#287]
## Describe the bug

When attempting to open the GUI application, it immediately quits, with the error "dndhandler quit unexpectedly".

### To Reproduce
1. Open the application, either directly from the disk image or from anywhere else the application has been moved to.

### Expected behavior

The app should open normally.

### Attachments

Here's the crash report:
```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               dndhandler [82876]
Path:                  /Applications/ChordPro.app/Contents/MacOS/dndhandler
Identifier:            org.chordpro.chordpro
Version:               0.977.13.1
Code Type:             X86-64 (Translated)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-06-06 00:32:53.3086 -0400
OS Version:            macOS 13.4 (22F66)
Report Version:        12
Anonymous UUID:        D8D51308-FF6F-D109-490C-6E8E698A1AC5

Sleep/Wake UUID:       E8DACA28-F7C2-4404-A7FB-BAA605387B43

Time Awake Since Boot: 150000 seconds
Time Since Wake:       8373 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: @executable_path/libtiff.5.dylib
Referenced from: <CDEEC624-5B38-334F-9925-A22593C92CAD> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_xrc-3.0.dylib
Reason: tried: '/Applications/ChordPro.app/Contents/MacOS/libtiff.5.dylib' (no such file), '/usr/local/lib/libtiff.5.dylib' (no such file), '/usr/lib/libtiff.5.dylib' (no such file, not in dyld cache)Library not loaded: @executable_path/libtiff.5.dylib
Referenced from: <82EC2BCA-6160-3EF3-A70C-5CA3A761843D> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_html-3.0.dylib
Reason: tried: '/Applications/ChordPro.app/Contents/MacOS/libtiff.5.dylib' (no such file), '/usr/local/lib/libtiff.5.dylib' (no such file), '/usr/lib/libtiff.5.dylib' (no such file, not in dyld cache)Library not loaded: @executable_path/libtiff.5.dylib
Referenced from: <02650996-7447-34B3-92D5-775102674BB0> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_qa-3.0.dylib
Reason: tried: '/Applications/ChordPro.app/Contents/MacOS/libtiff
(terminated at launch; ignore backtrace)

Thread 0 Crashed:
0   Rosetta Runtime Routines	       0x1020d19a8 ???
1   <translation info unavailable>	       0x10215f34c ???
2   dyld                          	       0x2022f8fd7 abort_with_payload_wrapper_internal + 82
3   dyld                          	       0x2022f9009 abort_with_payload + 9
4   dyld                          	       0x20227d8f0 dyld4::halt(char const*) + 375
5   dyld                          	       0x20227ab71 dyld4::prepare(dyld4::APIs&, dyld3::MachOAnalyzer const*) + 4526
6   dyld                          	       0x2022793bd start + 1805

Thread 1:: com.apple.rosetta.exceptionserver
0   runtime                       	    0x7ff7ffd28694 0x7ff7ffd24000 + 18068


Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000006   x1: 0x0000000000000001   x2: 0x0000000308402000   x3: 0x000000000000007f
    x4: 0x0000000308401c00   x5: 0x0000000000000000   x6: 0x00007ffca043d4e0   x7: 0x0000002000000000
    x8: 0x0000000308401c00   x9: 0x0000000000000000  x10: 0x000000000000007f  x11: 0x0000000308401c00
   x12: 0x000000000000007f  x13: 0x0000000308402000  x14: 0x0000000000000001  x15: 0x0000000000000006
   x16: 0x0000000000000209  x17: 0x0000000000000000  x18: 0x800000010a136b80  x19: 0x00000001020c7000
   x20: 0x00000000000007ff  x21: 0x000000010a13bfa0  x22: 0xffffffffffffffff  x23: 0x00000001020d197c
   x24: 0x00000002022f8fd7  x25: 0x00000001020d1248  x26: 0x000000010215d598  x27: 0x0000000000000000
   x28: 0x00000002022dec42   fp: 0x000000002c000000   lr: 0x000000010215f34c
    sp: 0x000000010a136b80   pc: 0x00000001020d19a8 cpsr: 0x80001000
   far: 0x000000010215f000  esr: 0x56000080  Address size fault

Binary Images:
       0x202273000 -        0x20230efff dyld (*) <9e98a840-a3ac-31c1-ab97-829af9bd6864> /usr/lib/dyld
    0x7ff7ffd24000 -     0x7ff7ffd53fff runtime (*) <c4d322c3-30c5-3c52-adb8-2fd12ec92bc0> /usr/libexec/rosetta/runtime
       0x10a0d1000 -        0x10a124fff libRosettaRuntime (*) <c8f25cdc-b334-3f67-bc30-50e9852f8da2> /Library/Apple/*/libRosettaRuntime
       0x102091000 -        0x102098fff org.chordpro.chordpro (*) <98f2adfc-da76-38b2-beba-ca39f078002d> /Applications/ChordPro.app/Contents/MacOS/dndhandler
       0x10a721000 -        0x10a790fff libwx_osx_cocoau_xrc-3.0.dylib (*) <cdeec624-5b38-334f-9925-a22593c92cad> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_xrc-3.0.dylib
       0x10a899000 -        0x10a8fcfff libwx_osx_cocoau_html-3.0.dylib (*) <82ec2bca-6160-3ef3-a70c-5ca3a761843d> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_html-3.0.dylib
       0x10a699000 -        0x10a6b0fff libwx_osx_cocoau_qa-3.0.dylib (*) <02650996-7447-34b3-92d5-775102674bb0> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_qa-3.0.dylib
       0x10abba000 -        0x10ac69fff libwx_osx_cocoau_adv-3.0.dylib (*) <aa159c1d-62f8-3550-aa10-23f04932813b> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_adv-3.0.dylib
       0x10b311000 -        0x10b5c0fff libwx_osx_cocoau_core-3.0.dylib (*) <76d5f1de-810c-3071-8798-1f590c597476> /Applications/ChordPro.app/Contents/MacOS/libwx_osx_cocoau_core-3.0.dylib
       0x10aa09000 -        0x10aa14fff libwx_baseu_xml-3.0.dylib (*) <d63b4fd5-5509-3f5e-800b-f31af286643c> /Applications/ChordPro.app/Contents/MacOS/libwx_baseu_xml-3.0.dylib
       0x10aa92000 -        0x10aab5fff libwx_baseu_net-3.0.dylib (*) <9f13b75d-619b-3a81-85e1-1c955bd984f2> /Applications/ChordPro.app/Contents/MacOS/libwx_baseu_net-3.0.dylib
       0x10bc5e000 -        0x10bdadfff libwx_baseu-3.0.dylib (*) <fed3754b-fefc-3cdd-a5a3-c423aa4b87c3> /Applications/ChordPro.app/Contents/MacOS/libwx_baseu-3.0.dylib
       0x10ae82000 -        0x10af05fff libjpeg.8.dylib (*) <438034e2-c120-351e-80bd-8785053385a4> /Applications/ChordPro.app/Contents/MacOS/libjpeg.8.dylib
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=409.3M resident=0K(0%) swapped_out_or_unallocated=409.3M(100%)
Writable regions: Total=157.4M written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=157.4M(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Rosetta Arena                     4096K        2 
Rosetta Generic                    932K      230 
Rosetta IndirectBranch              32K        1 
Rosetta JIT                      128.0M        1 
Rosetta Return Stack                20K        2 
Rosetta Thread Context              20K        2 
Stack                             8176K        1 
Stack Guard                       56.0M        1 
VM_ALLOCATE                       15.9M        7 
VM_ALLOCATE (reserved)             128K       10         reserved VM address space (unallocated)
__DATA                            14.7M      284 
__DATA_CONST                      14.0M      182 
__DATA_DIRTY                       632K       95 
__FONT_DATA                        2352        1 
__LINKEDIT                       174.7M       18 
__OBJC_RO                         66.3M        1 
__OBJC_RW                         2010K        1 
__TEXT                           234.6M      306 
dyld private memory                260K        2 
mapped file                        4.2G     1300 
===========                     =======  ======= 
TOTAL                              4.9G     2447 
TOTAL, minus reserved VM space     4.9G     2447 



-----------
Full Report
-----------

{"app_name":"dndhandler","timestamp":"2023-06-06 00:32:53.00 -0400","app_version":"","slice_uuid":"98f2adfc-da76-38b2-beba-ca39f078002d","build_version":"0.977.13.1","platform":1,"bundleID":"org.chordpro.chordpro","share_with_app_devs":1,"is_first_party":0,"bug_type":"309","os_version":"macOS 13.4 (22F66)","roots_installed":0,"name":"dndhandler","incident_id":"69AA4FE5-4AD7-4808-869B-DC2DC5E4E8C3"}
{
  "uptime" : 150000,
  "procRole" : "Default",
  "version" : 2,
  "userID" : 501,
  "deployVersion" : 210,
  "modelCode" : "MacBookPro17,1",
  "coalitionID" : 15502,
  "osVersion" : {
    "train" : "macOS 13.4",
    "build" : "22F66",
    "releaseType" : "User"
  },
  "captureTime" : "2023-06-06 00:32:53.3086 -0400",
  "incident" : "69AA4FE5-4AD7-4808-869B-DC2DC5E4E8C3",
  "pid" : 82876,
  "translated" : true,
  "cpuType" : "X86-64",
  "roots_installed" : 0,
  "bug_type" : "309",
  "procLaunch" : "2023-06-06 00:32:53.2404 -0400",
  "procStartAbsTime" : 3646566602371,
  "procExitAbsTime" : 3646568231174,
  "procName" : "dndhandler",
  "procPath" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/dndhandler",
  "bundleInfo" : {"CFBundleVersion":"0.977.13.1","CFBundleIdentifier":"org.chordpro.chordpro"},
  "storeInfo" : {"deviceIdentifierForVendor":"048AD720-B360-573E-B33B-B644FCBEE0A6","thirdParty":true},
  "parentProc" : "launchd",
  "parentPid" : 1,
  "coalitionName" : "org.chordpro.chordpro",
  "crashReporterKey" : "D8D51308-FF6F-D109-490C-6E8E698A1AC5",
  "codeSigningID" : "",
  "codeSigningTeamID" : "",
  "codeSigningValidationCategory" : 0,
  "codeSigningTrustLevel" : 0,
  "wakeTime" : 8373,
  "sleepWakeUUID" : "E8DACA28-F7C2-4404-A7FB-BAA605387B43",
  "sip" : "enabled",
  "exception" : {"codes":"0x0000000000000000, 0x0000000000000000","rawCodes":[0,0],"type":"EXC_CRASH","signal":"SIGABRT"},
  "termination" : {"code":1,"flags":518,"namespace":"DYLD","indicator":"Library missing","details":["(terminated at launch; ignore backtrace)"],"reasons":["Library not loaded: @executable_path\/libtiff.5.dylib","Referenced from: <CDEEC624-5B38-334F-9925-A22593C92CAD> \/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_xrc-3.0.dylib","Reason: tried: '\/Applications\/ChordPro.app\/Contents\/MacOS\/libtiff.5.dylib' (no such file), '\/usr\/local\/lib\/libtiff.5.dylib' (no such file), '\/usr\/lib\/libtiff.5.dylib' (no such file, not in dyld cache)Library not loaded: @executable_path\/libtiff.5.dylib","Referenced from: <82EC2BCA-6160-3EF3-A70C-5CA3A761843D> \/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_html-3.0.dylib","Reason: tried: '\/Applications\/ChordPro.app\/Contents\/MacOS\/libtiff.5.dylib' (no such file), '\/usr\/local\/lib\/libtiff.5.dylib' (no such file), '\/usr\/lib\/libtiff.5.dylib' (no such file, not in dyld cache)Library not loaded: @executable_path\/libtiff.5.dylib","Referenced from: <02650996-7447-34B3-92D5-775102674BB0> \/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_qa-3.0.dylib","Reason: tried: '\/Applications\/ChordPro.app\/Contents\/MacOS\/libtiff"]},
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "faultingThread" : 0,
  "threads" : [{"triggered":true,"id":3302765,"threadState":{"x":[{"value":6},{"value":1},{"value":13023322112},{"value":127},{"value":13023321088},{"value":0},{"value":140722997286112},{"value":137438953472},{"value":13023321088},{"value":0},{"value":127},{"value":13023321088},{"value":127},{"value":13023322112},{"value":1},{"value":6},{"value":521},{"value":0},{"value":9223372041318787968},{"value":4329336832},{"value":2047},{"value":4464033696},{"value":18446744073709551615},{"value":4329380220},{"value":8626606039,"symbolLocation":82,"symbol":"abort_with_payload_wrapper_internal"},{"value":4329378376},{"value":4329952664},{"value":0},{"value":8626498626,"symbolLocation":10,"symbol":"__abort_with_payload"}],"flavor":"ARM_THREAD_STATE64","lr":{"value":4329960268},"cpsr":{"value":2147487744},"fp":{"value":738197504},"sp":{"value":4464012160},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":4329380264,"matchesCrashFrame":1},"far":{"value":4329959424}},"frames":[{"imageOffset":4329380264,"region":"Rosetta Runtime Routines","symbolLocation":10664,"imageIndex":13},{"imageOffset":4329960268,"region":"<translation info unavailable>","imageIndex":13},{"imageOffset":548823,"symbol":"abort_with_payload_wrapper_internal","symbolLocation":82,"imageIndex":0},{"imageOffset":548873,"symbol":"abort_with_payload","symbolLocation":9,"imageIndex":0},{"imageOffset":43248,"symbol":"dyld4::halt(char const*)","symbolLocation":375,"imageIndex":0},{"imageOffset":31601,"symbol":"dyld4::prepare(dyld4::APIs&, dyld3::MachOAnalyzer const*)","symbolLocation":4526,"imageIndex":0},{"imageOffset":25533,"symbol":"start","symbolLocation":1805,"imageIndex":0}]},{"id":3302772,"name":"com.apple.rosetta.exceptionserver","frames":[{"imageOffset":18068,"imageIndex":1}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 8626057216,
    "size" : 638976,
    "uuid" : "9e98a840-a3ac-31c1-ab97-829af9bd6864",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 140703125618688,
    "size" : 196608,
    "uuid" : "c4d322c3-30c5-3c52-adb8-2fd12ec92bc0",
    "path" : "\/usr\/libexec\/rosetta\/runtime",
    "name" : "runtime"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4463595520,
    "size" : 344064,
    "uuid" : "c8f25cdc-b334-3f67-bc30-50e9852f8da2",
    "path" : "\/Library\/Apple\/*\/libRosettaRuntime",
    "name" : "libRosettaRuntime"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4329115648,
    "CFBundleIdentifier" : "org.chordpro.chordpro",
    "size" : 32768,
    "uuid" : "98f2adfc-da76-38b2-beba-ca39f078002d",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/dndhandler",
    "name" : "dndhandler",
    "CFBundleVersion" : "0.977.13.1"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4470214656,
    "size" : 458752,
    "uuid" : "cdeec624-5b38-334f-9925-a22593c92cad",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_xrc-3.0.dylib",
    "name" : "libwx_osx_cocoau_xrc-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4471754752,
    "size" : 409600,
    "uuid" : "82ec2bca-6160-3ef3-a70c-5ca3a761843d",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_html-3.0.dylib",
    "name" : "libwx_osx_cocoau_html-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4469657600,
    "size" : 98304,
    "uuid" : "02650996-7447-34b3-92d5-775102674bb0",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_qa-3.0.dylib",
    "name" : "libwx_osx_cocoau_qa-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4475035648,
    "size" : 720896,
    "uuid" : "aa159c1d-62f8-3550-aa10-23f04932813b",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_adv-3.0.dylib",
    "name" : "libwx_osx_cocoau_adv-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4482732032,
    "size" : 2818048,
    "uuid" : "76d5f1de-810c-3071-8798-1f590c597476",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_osx_cocoau_core-3.0.dylib",
    "name" : "libwx_osx_cocoau_core-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4473262080,
    "size" : 49152,
    "uuid" : "d63b4fd5-5509-3f5e-800b-f31af286643c",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_baseu_xml-3.0.dylib",
    "name" : "libwx_baseu_xml-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4473823232,
    "size" : 147456,
    "uuid" : "9f13b75d-619b-3a81-85e1-1c955bd984f2",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_baseu_net-3.0.dylib",
    "name" : "libwx_baseu_net-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4492484608,
    "size" : 1376256,
    "uuid" : "fed3754b-fefc-3cdd-a5a3-c423aa4b87c3",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libwx_baseu-3.0.dylib",
    "name" : "libwx_baseu-3.0.dylib"
  },
  {
    "source" : "P",
    "arch" : "x86_64",
    "base" : 4477952000,
    "size" : 540672,
    "uuid" : "438034e2-c120-351e-80bd-8785053385a4",
    "path" : "\/Applications\/ChordPro.app\/Contents\/MacOS\/libjpeg.8.dylib",
    "name" : "libjpeg.8.dylib"
  },
  {
    "size" : 0,
    "source" : "A",
    "base" : 0,
    "uuid" : "00000000-0000-0000-0000-000000000000"
  }
],
  "sharedCache" : {
  "base" : 140703594561536,
  "size" : 21474836480,
  "uuid" : "9bdf650c-7fc0-3de1-bac4-8535aea9b6aa"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=409.3M resident=0K(0%) swapped_out_or_unallocated=409.3M(100%)\nWritable regions: Total=157.4M written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=157.4M(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nRosetta Arena                     4096K        2 \nRosetta Generic                    932K      230 \nRosetta IndirectBranch              32K        1 \nRosetta JIT                      128.0M        1 \nRosetta Return Stack                20K        2 \nRosetta Thread Context              20K        2 \nStack                             8176K        1 \nStack Guard                       56.0M        1 \nVM_ALLOCATE                       15.9M        7 \nVM_ALLOCATE (reserved)             128K       10         reserved VM address space (unallocated)\n__DATA                            14.7M      284 \n__DATA_CONST                      14.0M      182 \n__DATA_DIRTY                       632K       95 \n__FONT_DATA                        2352        1 \n__LINKEDIT                       174.7M       18 \n__OBJC_RO                         66.3M        1 \n__OBJC_RW                         2010K        1 \n__TEXT                           234.6M      306 \ndyld private memory                260K        2 \nmapped file                        4.2G     1300 \n===========                     =======  ======= \nTOTAL                              4.9G     2447 \nTOTAL, minus reserved VM space     4.9G     2447 \n",
  "legacyInfo" : {
  "threadTriggered" : {

  }
},
  "logWritingSignature" : "e0e8625044b4fb6daf4465cb4ba782178544610c",
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "63582c5f8a53461413999550",
      "factorPackIds" : {

      },
      "deploymentId" : 240000002
    },
    {
      "rolloutId" : "5f72dc58705eff005a46b3a9",
      "factorPackIds" : {

      },
      "deploymentId" : 240000015
    }
  ],
  "experiments" : [
    {
      "treatmentId" : "c28e4ee6-1b08-4f90-8e05-2809e78310a3",
      "experimentId" : "6317d2003d24842ff850182a",
      "deploymentId" : 400000013
    },
    {
      "treatmentId" : "6dd670af-0633-45e4-ae5f-122ae4df02be",
      "experimentId" : "64406ba83deb637ac8a04419",
      "deploymentId" : 900000005
    }
  ]
}
}
```

## System information
 - OS:  macOS Ventura 13.4
 - Where did you get ChordPro from?  GitHub

 - ChordPro version:  Can't tell
 - Perl version: 5.30.3

**Comment by sciurius** [comment]
> Version:               0.977.13.1

That doesn't sound like a recent version.

> OS: macOS Ventura 13.4

ChordPro is built for macOS Catalina so it could be that some or your libraries are too old.
Can you download and try https://www.squirrel.nl/pub/xfer/uploads/3CSb2ZC6gcVbFwKKciHyKfBg.010_006.dmg ?

**Comment by wtimme** [comment]
Thanks, @sciurius! I've downloaded the DMG you provided and was able to open it just fine. I'm running macOS 13.4.1 (22F82).

**Comment by jlg89** [comment]
Same issue here, the @sciurius version above opens fine.

Ventura 13.4.1 (C) on Apple M1

(so this could be an ARM issue)

**Comment by sciurius** [comment]
The temporary fix of 6.010_006 will be part of the next release, hoping this will prevent more problems.

I'd still need support of a macOS developer to get these (and other macOS specific) issues sorted out.
